AuctionatorSearch.lua

```diff
@@ -1,9 +1,9 @@
-Auctionator.Search = {
+Auctionator.OldSearch = {
```

Comment:
I'm OK with removing this file in its entirety if we no longer need it for reference for our re-implementation.